Published weekly changelog (#440)
CoboGlobal/product-manual: en/portal/changelog.mdx

description: "Stay updated with the latest documentation changes for Cobo Portal."
---

<Update label="2025/03/05 - 2025/03/11">
**Risk Controls**

> - Updated message signing policy documentation in [Manage message signing policy](/en/portal/risk-controls/off-chain-tx-manage-message-signing) and [Set up message signing policy](/en/portal/risk-controls/off-chain-tx-set-message-signing)

Reviewer: ```suggestion
- Added message signing policy documentation: [Manage message signing policy](/en/portal/risk-controls/off-chain-tx-manage-message-signing) and [Set up message signing policy](/en/portal/risk-controls/off-chain-tx-set-message-signing)
```

> - Enhanced Safe risk controls documentation in [Set up risk controls for Safe{Wallet} message signing](/en/portal/risk-controls/safe-risk-controls)

Reviewer: ```suggestion
- Added Safe&#123;Wallet&#125; risk controls documentation: [Set up risk controls for Safe&#123;Wallet&#125; message signing](/en/portal/risk-controls/safe-risk-controls)
```

> - Updated risk control terminology in [Introduction to Risk Controls](/en/portal/risk-controls/introduction) to align with UI

Reviewer: ```suggestion
```

**MPC Wallets**

> - Updated [Introduction to Auto-Sweep](/en/portal/mpc-wallets/ocw/auto-sweep-overview) with revised fee information and clarified supported chain details

Reviewer 2: ```suggestion
Updated [Introduction to Auto-Sweep](/en/portal/mpc-wallets/ocw/auto-sweep-overview) with clarified supported chain details
```

- Aligned terminology in [Enable Auto-Sweep](/en/portal/mpc-wallets/ocw/enable-auto-sweep) with UI updates

**Cobo Guard**
- Enhanced [Generate and back up your key shares](/en/guard/generate-and-back-up-key-shares) with key share backup file location information

**Token Bridge**
- Added Solana to supported USDT Bridge in [Supported assets](/en/apps/bridge/how-to-use/manage/supported-assets)

**Screening**
- Updated English Screening documentation with comprehensive information about CipherOwl integration and risk assessment in:
  - [Introduction to Screening](/en/apps/screening/introduction)
  - [Manage post-screening policies](/en/apps/screening/risk-policies)
  - [Set up Screening](/en/apps/screening/set-up)
  - [Manage Screening results](/en/apps/screening/viewscreening)
- Removed outdated Chinese Screening documentation

**Staking**

> - Updated staking documentation in [Stake](/en/apps/staking/stake) with latest features and processes

Reviewer: ```suggestion
- Updated [Stake](/en/apps/staking/stake) to add ETH staking overview and optimize step-by-step information for better accuracy
```
